prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-emails-container/notification-emails-container.component.ts

```diff
@@ -30,10 +32,23 @@ export class NotificationEmailsContainerComponent implements OnInit {
 
   public getTemplateProperties(template: EmailTemplate) {
     return [
+      {
+        key: 'Subject',
+        value: template.subject
+      },
```

Comment:
Is there a reason on the card view, the order of fields is different than the detail view?

Reply:
no specific reason. Updated the card view to match the detail view.